anvio/workflows/contigs/__init__.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, args=None, run=terminal.Run(), progress=terminal.Progress()):
                                     "anvi_run_kegg_kofams": {"run": True, "threads": 4},
                                     "anvi_run_ncbi_cogs": {"run": True, "threads": 5},
                                     "anvi_run_scg_taxonomy": {"run": True, "threads": 6},
-                                    'anvi_run_trna_scan': {"run": True, "threads": 6},
```

Comment:
Shouldn't you also add `'--also-scan-trnas': True` to `anvi-run-hmms` rule so the functionality is retained in resulting contigs databases?